### ISSUE TITLE
Example code issue

### ISSUE THREAD

**Issue by SungmanHong** [faustomorales/keras-ocr#114]
Hi

I'm just using
your
https://keras-ocr.readthedocs.io/en/latest/examples/end_to_end_training.html

Example of end_to_end_training

And
your provided colab example of
https://colab.research.google.com/drive/19dGKong-LraUG3wYlJuPCquemJ13NN8R

There is some issue of(some mistypo(?))

1. In CELL [7]

```
detector = keras_ocr.detection.Detector(weights='clovaai_general')
recognizer = keras_ocr.recognition.Recognizer
    alphabet=recognizer_alphabet,
    weights='kurapan',
    include_top=False
)
```
should be

```
detector = keras_ocr.detection.Detector(weights='clovaai_general')
recognizer = keras_ocr.recognition.Recognizer(
    alphabet=recognizer_alphabet,
    weights='kurapan'
)
```

2. In cell [13]

```
pipeline = keras_ocr.pipelines.Pipeline(detector=detector, recognizer=recognizer)
image, lines = next(image_generators[0])
predictions = pipeline.recognize(images=[image])[0]
drawn = keras_ocr.tools.drawBoxes(
    image=image, boxes=predictions, boxes_format='predictions'
)
print(
    'Actual:', '\n'.join([' '.join([character for _, character in line]) for line in lines]),
    'Predicted:', [text for text, box in predictions])
plt.imshow(drawn)
```
should be

```
pipeline = keras_ocr.pipeline.Pipeline(detector=detector, recognizer=recognizer)
image, lines = next(image_generators[0])
predictions = pipeline.recognize(images=[image])[0]
drawn = keras_ocr.tools.drawBoxes(
    image=image, boxes=predictions, boxes_format='predictions'
)
print(
    'Actual:', '\n'.join([' '.join([character for _, character in line]) for line in lines]),
    'Predicted:', [text for text, box in predictions])
plt.imshow(drawn)
```

thanks.

**Comment by SungmanHong** [comment]
I'm not sure about it...but 
when my len(alphabet) >= 800
The predicted is tend to ''?

Actual: F
o r   d i s c o v
e r i n g   o f   v a
s a   p a r r o t
s   k e p t   i n Predicted: ['', 's', '', 'a', '', '', 'e', 'a', 'sa', '', 'et', '']

I think because len of alphabet is too high -> too low to find correct alphabet -> being tend to '' to reduce loss function?
If I manually test new loss funnction that penalty the ''?

**Comment by SungmanHong** [comment]
Hi, It is being solved when I train large epoch.... thank you!
